roles/php56fpm/tasks/main.yml

```diff
@@ -21,7 +21,6 @@
   - php-cli
   - php-pecl-apcu
   - php-curl
-  - php-ldap
```

Comment:
The current engineblock still needs php-ldap. We'll remove it after the EB release without ldap is released